include/enum_quda_fortran.h

```diff
@@ -41,9 +41,8 @@
 #define QUDA_QDPJIT_GAUGE_ORDER 11     // expect *gauge[4] even-odd spacetime row-column color
 #define QUDA_CPS_WILSON_GAUGE_ORDER 12 // expect *gauge even-odd spacetime column-row color
 #define QUDA_MILC_GAUGE_ORDER 13       // expect *gauge even-odd mu spacetime row-column order
-#define QUDA_MILC_SITE_GAUGE_ORDER                                                                                     \
```

Comment:
can you remove this new line here, else this will break Fortran interface?

Reply:
(1) In the 2nd commit, I added the   UKQCD <-> Dirac-Pauli routines and tested them.
(2) Yes, I have tested all of the added routines with quda_laph.  I will still do additional tests to compare to chroma today.
(3) I restored that original line in the fortran enum file.
(4) I'll do the clang formatting asap.